include/boost/math/tools/polynomial.hpp

```diff
@@ -418,7 +418,7 @@ class polynomial
 #endif
    }
 
-   polynomial<T> integrate() const
+   BOOST_MATH_GPU_ENABLED polynomial<T> integrate() const
```

Comment:
Question: can we actually use this on a device - with std::vector?

Reply:
Looks like no: https://github.com/NVIDIA/cccl/discussions/1032